addons/wiki/Dockerfile

```diff
@@ -1,6 +1,9 @@
 FROM node:0.12
 
-RUN mkdir -p /code
+RUN mkdir -p /code \
+    && chown node:node /code
+
+USER node
```

Comment:
i'd recommend `gosu` since this it hard to ever get root on the container once its running, which can be useful for apt-get installs and tools.